spec/features/edit_scheduled_messages_spec.rb

```diff
@@ -15,4 +15,24 @@
     expect(page).to have_content "Scheduled message updated successfully"
     expect(page).to have_content new_title
   end
+
+  scenario "from list of scheduled messages with tags" do
+    old_title = "Old title"
+    new_title = "New title"
+    tags = "tag_one, tag_two, tag_three"
+    create(:scheduled_message, title: old_title)
+
+    login_with_oauth
+    visit scheduled_messages_path
+    click_on "Edit"
+    fill_in "Title", with: new_title
+    fill_in "Tags (seperated by commas)", with: tags
```

Comment:
The tags get added as a single string, which I'm not exactly comfortable with just leaving, so I am going to work on improving that a bit.